data/migrations/V0053__add_ofec_filings_all_mv.sql

```diff
@@ -0,0 +1,481 @@
+            rfai_filings.previous_file_number,
+            rfai_filings.report_type_full,
+            rfai_filings.primary_general_indicator,
+            rfai_filings.request_type,
+            rfai_filings.amendment_indicator,
+            rfai_filings.update_date,
+            rfai_filings.pdf_url,
+            rfai_filings.means_filed,
+            rfai_filings.html_url,
+            rfai_filings.fec_url,
+            rfai_filings.amendment_chain,
+            rfai_filings.most_recent_file_number,
+            rfai_filings.is_amended,
+            rfai_filings.most_recent,
+            rfai_filings.amendement_version,
+            rfai_filings.state,
+            rfai_filings.office,
+            rfai_filings.district,
+            rfai_filings.party,
+            rfai_filings.cmte_tp,
+            rfai_filings.office_cmte_tp
+           FROM rfai_filings
+        )
+ SELECT row_number() OVER () AS idx,
+    combined.candidate_id,
+    combined.candidate_name,
+    combined.committee_id,
+    combined.committee_name,
+    combined.sub_id,
+    combined.coverage_start_date,
+    combined.coverage_end_date,
+    combined.receipt_date,
+    combined.election_year,
+    combined.form_type,
+    combined.report_year,
+    combined.cycle,
+    combined.report_type,
+    combined.document_type,
+    combined.document_type_full,
+    combined.beginning_image_number,
+    combined.ending_image_number,
+    combined.pages,
+    combined.total_receipts,
+    combined.total_individual_contributions,
+    combined.net_donations,
+    combined.total_disbursements,
+    combined.total_independent_expenditures,
+    combined.total_communication_cost,
+    combined.cash_on_hand_beginning_period,
+    combined.cash_on_hand_end_period,
+    combined.debts_owed_by_committee,
+    combined.debts_owed_to_committee,
+    combined.house_personal_funds,
+    combined.senate_personal_funds,
+    combined.opposition_personal_funds,
+    combined.treasurer_name,
+    combined.file_number,
+    combined.previous_file_number,
+    combined.report_type_full,
+    combined.primary_general_indicator,
+    combined.request_type,
+    combined.amendment_indicator,
+    combined.update_date,
+    combined.pdf_url,
+    combined.means_filed,
+    combined.html_url,
+    combined.fec_url,
+    combined.amendment_chain,
+    combined.most_recent_file_number,
+    combined.is_amended,
+    combined.most_recent,
+    combined.amendment_version,
+    combined.state,
+    combined.office,
+    combined.district,
+    combined.party,
+    combined.cmte_tp,
+    combined.office_cmte_tp
+   FROM combined
+WITH DATA;
+
+ALTER TABLE public.ofec_filings_all_mv
+  OWNER TO fec;
+GRANT ALL ON TABLE public.ofec_filings_all_mv TO fec;
+GRANT SELECT ON TABLE public.ofec_filings_all_mv TO fec_read;
+
+
+CREATE INDEX ofec_filings_all_mv_amendment_indicator_idx_idx1
+  ON public.ofec_filings_all_mv
+  USING btree
+  (amendment_indicator COLLATE pg_catalog."default", idx);

```

Comment:
Why we are including the identifier in all the indexes? Maybe I am missing someting

Reply:
Did some research and in itemized sorts the unique key is always appended to the sort for consistent sorting in case of a tie. This only happens when the resource inherits from the ItemizedResource class:
https://github.com/18F/openFEC/blob/8e39a1f8e04915ade2e97898b96eda2ea686e906/webservices/common/views.py#L105

In this case, the filings resource inherits from the ApiBase and not Itemized, to the compound indexes won't help.
https://github.com/18F/openFEC/blob/8e39a1f8e04915ade2e97898b96eda2ea686e906/webservices/resources/filings.py#L20

This existed before this PR, but we can clean it up since we are making changes in here anyway. Thanks @fecjjeng and @pkfec